src/test/java/marquez/api/NamespaceIntegrationTest.java

```diff
@@ -33,6 +33,26 @@ public void testCreateNamespace() {
     assertThat(responseBody.getNamespace().getDescription()).isEqualTo(DESCRIPTION);
   }
 
+  @Test
+  public void testCreateNamespace_NoDup() {
+    APP.client()
+        .target(URI.create("http://localhost:" + APP.getLocalPort()))
+        .path("/namespaces/" + NAMESPACE_NAME)
+        .request(MediaType.APPLICATION_JSON)
+        .put(Entity.json(createNamespaceRequest));
+    Response res =
+        APP.client()
+            .target(URI.create("http://localhost:" + APP.getLocalPort()))
+            .path("/namespaces/" + NAMESPACE_NAME)
+            .request(MediaType.APPLICATION_JSON)
+            .put(Entity.json(createNamespaceRequest));
+    CreateNamespaceResponse responseBody = res.readEntity(CreateNamespaceResponse.class);
+    assertEquals(Response.Status.OK.getStatusCode(), res.getStatus());
+    assertThat(responseBody.getNamespace().getCreatedAt()).isAfter(START_TIME);
```

Comment:
We'll want to discuss which testing framework we'd like to use for consistency across tests. I think there's a lot we can do with the simplicity of `org.junit.Assert` :)